src/Land/Model/land_bc.jl

```diff
@@ -95,6 +96,40 @@ function SurfaceDrivenWaterBoundaryConditions(
     return SurfaceDrivenWaterBoundaryConditions{FT, typeof.(args)...}(args...)
 end
 
+"""
+    SurfaceDrivenHeatBoundaryConditions{FT, SWD} <: AbstractBoundaryConditions
+
+Boundary condition type to be used when the user wishes to 
+apply physical fluxes of heat at the top of the domain 
+(according to radiative energy fluxes).
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct SurfaceDrivenHeatBoundaryConditions{FT, SWD} <:
+       AbstractBoundaryConditions where {FT, SWD}
+    "Net short wave flux model"
+    nswf_model::SWD
+end
+
+"""
+    SurfaceDrivenHeatBoundaryConditions(
+        ::Type{FT};
+        nswf_model::AbstractNSWfluxModel{FT} = NSWflux_from_albedo{FT}()
+    ) where {FT}
+
+Constructor for the SurfaceDrivenWaterBoundaryConditions object. The default
+is a constant precipitation rate on the subgrid scale, and no runoff.
+"""
+function SurfaceDrivenHeatBoundaryConditions(
+    ::Type{FT};
+    nswf_model::AbstractNetSwFluxModel{FT} = prescribed_net_sw_flux{FT}(
```

Comment:
does this work? The name is `PrescribedNetSwFlux`, no? probably good to add a unit test creating an instance of SurfaceDrivenHeatBoundaryConditions

Reply:
DONE :)